changes/process_changes.py

```diff
@@ -0,0 +1,124 @@
+# coding: utf-8
```

Comment:
Beyond testcases, none of our source files contain any non-ASCII character, so I would be in favor of removing this.

Reply:
OK